modules/altinnsalary/view/altinn1list.php

```diff
@@ -5,11 +5,16 @@
 $query_altinn = "select * from $db_table order by AltinnReport1ID";
 $result = $_lib['db']->db_query($query_altinn);
 
+$salary_workrelation_array = array();
```

Comment:
Try to move all js related stuff to one file and unify the php vars needed for it